app/src/main/java/com/example/rominakehl/bildung_4_0/AndroidStackViewActivity.java

```diff
@@ -0,0 +1,35 @@
+package com.example.rominakehl.bildung_4_0;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.widget.StackView;
+
+import java.util.ArrayList;
+
+public class AndroidStackViewActivity extends Activity {
+
+
+    private static StackView stackView;
+    private static ArrayList<StackItems> list;
+
+    private static final Integer[] icons = {R.drawable.messen, R.drawable.unterricht2,
```

Comment:
When you rename the jpgs, use `git mv <old-file> <new-file>` so the and rename is happening and not a delet and new adding of the jpgs.